### PR TITLE
Nikon PictureControl directory extraction

### PR DESCRIPTION
**Pull request by gtiwari333**
This PR ports the change that were made on C# repo to this Java repo. 
https://github.com/drewnoakes/metadata-extractor-dotnet/pull/341/files

### CODE REVIEW

**Comment by gtiwari333** [comment]
@drewnoakes  please review and merge/release this.

 The image files that i used are from https://github.com/drewnoakes/metadata-extractor-images/wiki/ContentSummary. Let me know if there's an easier way to crop/resize those images to low resolution and preserve all the metadata.

File: Tests/com/drew/metadata/exif/NikonType2MakernoteTest2.java [hunk 1]
```diff
@@ -56,6 +61,30 @@ public void setUp() throws Exception
         assertNotNull(_exifSubIFDDirectory);
     }
 
+
+    @Test
+    public void testNikonPictureControl1Directory() throws Exception {
```
Comment (drewnoakes) [inline Tests/com/drew/metadata/exif/NikonType2MakernoteTest2.java:66]: We don't add images to this repo. We have a separate repo for regression testing with gigabytes of images. Please remove these tests. I can run the regression tests if you don't want to go down that rabbit hole.

**Review by drewnoakes — COMMENTED** [review]
Looks great, thanks!

File: Tests/com/drew/metadata/exif/NikonType2MakernoteTest2.java (hunk 1 above)
Comment (gtiwari333) [reply Tests/com/drew/metadata/exif/NikonType2MakernoteTest2.java]: Done @drewnoakes

**Comment by drewnoakes** [comment]
Please remove all test data and unit tests and squash history. We use the regression test suite for this as it's more comprehensive, has better ergonomics, and allows comparison between Java and .NET implementations.

**Comment by gtiwari333** [comment]
Will open new PR with squashed commits

**Pull request closed without merging**